OnlineStore/OnlineStore.DAL/ApplicationDbContext/AppDbContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore;
+using OnlineStore.DAL.EntitiesConfiguration;
+using OnlineStore.Domain.Entities;
+
+namespace OnlineStore.DAL.ApplicationDbContext
+{
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options) { }
+
+        public DbSet<Product> Products { get; set; }
+        public DbSet<Category> Categories { get; set; }
+        public DbSet<Order> Orders { get; set; }
+        public DbSet<OrderProduct> OrderProducts { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(ProductConfiguration).Assembly);
```

Comment:
you can use this only one time. no necessary to Apply Configurations From Assembly for every configuration because they are already in one assembly)